.ruby-version

```diff
@@ -1 +1 @@
-2.4.1
+2.4.4
```

Comment:
Why do we need to update .ruby-version? Will this break anything?

Reply:
to add onto this, tests use ruby 2.5, should we use ruby 2.5 instead?

Reply:
this was me being inconsistent, the example config i'd copied from originally used the 2.4.4 ruby image, and so i tested it with that locally. I think it's better I change it back to 2.4.1 and use that image for tests as well